node_modules/gh-users/tests/test-calendar.js

```diff
@@ -412,6 +412,60 @@ describe('Users - calendars', function() {
                 });
             });
         });
+
+        /**
+         * Test that verifies the correct description is returned when getting a timetable calendar
+         */
+        it('verify the description when getting a timetable calendar', function(callback) {
+            TestsUtil.generateTestUsers(global.tests.apps.cam2013, 1, false, function(simon) {
+                UsersTestsUtil.assertGetMe(simon.client, function(me) {
+                    // Set up an event with lots of metadata
+                    var calendarStart = moment().subtract(1, 'day').format();
+                    var calendarEnd = moment().add(30, 'day').format();
+                    SeriesTestUtil.generateSerieWithEvents(simon.client, 1, 1, calendarStart, calendarEnd, function(series) {
+                        var serie = series[0];
+                        var event = serie.events[0];
+                        var type = 'Lab';
+                        var notes = 'Whee, notes';
+                        EventsTestsUtil.assertUpdateEvent(global.tests.admins.cam2013.client, event.id, {'notes': notes, 'type': type}, function(updatedEvent) {
+
+                            // Subscribe on the series
+                            SeriesTestUtil.assertSubscribeSeries(simon.client, serie.id, simon.profile.id, null, function() {
+
+                                // Get the iCal feed
+                                UsersTestsUtil.assertGetUserCalendarICal(simon.client, simon.profile.id, me.calendarToken, [updatedEvent], function(calendar) {
+
+                                    // There should only be 1 event in the calendar
+                                    assert.strictEqual(calendar.subComponents.length, 1);
+                                    var expectedDescription = 'Lecturer: ' + event.organisers.join(', ') + '\n';
+                                    expectedDescription += 'Type: ' + type + '\n\n';
+                                    expectedDescription += notes;
+                                    assert.strictEqual(calendar.subComponents[0].model.description, expectedDescription);
+
+                                    // Add another lecturer
```

Comment:
Should we test the various permutations for event descriptions (e.g., no organisers, 1 organiser, multiple organisers, type, no type, notes, no notes, etc.) ?